src/text-field-base.tsx

```diff
@@ -336,6 +336,18 @@ export const TextFieldBase = React.forwardRef<any, TextFieldBaseProps>(
                             </Text3>
                         </div>
                     )}
+                    {label && (
+                        <Label
+                            style={labelStyle}
+                            error={error}
+                            forId={id}
+                            inputState={inputState}
+                            shrinkLabel={shrinkLabel}
+                            optional={!rest.required}
+                        >
+                            {label}
+                        </Label>
+                    )}
```

Comment:
Moved the label before the input. This avoids a bug (I think it's a bug) in Safari/VoiceOver which made the label being read twice by screen reader (when focusing the input, and then as an independent element). This doesn't happen if the label appears before the input in the html.